gradle/build.go

```diff
@@ -17,7 +17,12 @@
 package gradle
 
 import (
+	"crypto/sha256"
+	"encoding/hex"
 	"fmt"
+	"github.com/paketo-buildpacks/libpak/bindings"
```

Comment:
By convention this should be grouped with the other external imports. We typically sort imports into three groups, with imports sorted alphabetically in each group.
1. standard library imports
2. external imports
3. internal imports (packages from the same go module)

You can configure Goland to do this for you, or you can do it with:
```
goimports -l -w -local github.com/paketo-buildpacks/gradle .
```

It would be good to enforce our formatting opinions in a GHA check (along with a basic `go fmt` for that matter!)

Reply:
This is something I'm not a fan of in Goland. VS Code would automatically do this when I saved files. I've added the filewatchers in Goland so it's doing this now. Thanks.